src/pages/Recipes/RecipeEdition/RecipeEdition.jsx

```diff
@@ -0,0 +1,95 @@
+import { yupResolver } from "@hookform/resolvers/yup";
+import React from "react";
+import { useForm } from "react-hook-form";
+import { Route, Routes, useNavigate } from "react-router";
+import { useParams } from "react-router-dom";
+import { v4 as uuidv4 } from "uuid";
+import { Text } from "@/components";
+import AddIngredient from "../AddIngredient/AddIngredient";
+import useRecipe from "../hooks/api/useRecipe";
+import useUpdateRecipe from "../hooks/api/useUpdateRecipe";
+import useIngredients from "../hooks/useIngredients";
+import { schema } from "../RecipeCreation/formSchema";
+import RecipeCreation from "../RecipeCreation/RecipeCreation";
+
+const RecipeEditionRouter = () => {
+  const params = useParams();
+  const recipeId = params.id;
+  const recipeDetails = useRecipe(recipeId).data;
```

Comment:
If we're suing hooks - it's most common to spread the results, instead of using `.` to get some property from returned object. Moreover, it's easier to extend that if we'd like to grab another property, for example `isLoading` - we could easily add it to the extracted object.

Moreover, if we'd like to name the property in some more meaningful way - we can always alias properties like:
```suggestion
  const { data: recipeDetails } = useRecipe(recipeId);
```